src/vm/exceptionhandling.cpp

```diff
@@ -6184,11 +6187,11 @@ bool ExceptionTracker::IsInStackRegionUnwoundBySpecifiedException(CrawlFrame * p
 
     // Remember that sfLowerBound and sfUpperBound are in the "OS format".
     // Refer to the comment for CallerStackFrame for more information.
-#if defined(_TARGET_AMD64_)
+#ifndef USE_FUNCLET_CALL_HELPER
```

Comment:
@janvorli FYI, I felt like the above due to the current implementation of ``GetCurrentSP`` (for x86). 

I checked the corresponding implementation in AMD64, and found that it is a bit different. 
(I submitted #9838 to fix this).